implementations/kmp-json-schema-validator/.gitattributes

```diff
@@ -0,0 +1,8 @@
+#
+# https://help.github.com/articles/dealing-with-line-endings/
+#
+# Linux start script should use lf
+/gradlew        text eol=lf
```

Comment:
Do we even need this file as there are no wrapper scripts in repo?

Reply:
Initially there was a wrapper but I have decided that it shouldn't be committed (other impls that using Gradle doesn't have it). But I forgot about this file